### ISSUE TITLE
Alt-tabbing out of p5.js in the middle of calling a function moves the text cursor to the end of the code document

### ISSUE THREAD

**Issue by jpark11** [processing/p5.js-web-editor#106]
Say I am writing ellipse(300, __ , __, ___)

and I'm referencing another document (say that p5.js book online) for the second parameter, alt-tabbing to the book and alt-tabbing back to the p5.js online editor moves the text cursor to the end of the code document instead of where I left off in the 2nd parameter

**Comment by catarak** [comment]
This is on Windows, right? Which browser are you using? I just tried this on Firefox and Chrome (on Windows) and could not reproduce, though there are known issues of using the editor with Edge (see #27).

**Comment by jpark11** [comment]
Ah I was using Chrome on a Mac

Jeff

On Mon, Sep 19, 2016 at 5:05 PM, Cassie Tarakajian <notifications@github.com

**Comment by catarak** [comment]
Okay, let me try again on Mac and see what happens

**Comment by catarak** [comment]
Just tried again on Mac and I'm not able to reproduce. By alt + tabbing do you mean Command + Tabbing? As far as I know, alt + tabbing isn't a keyboard shortcut on Mac.

**Comment by jpark11** [comment]
Oh yes. Sorry. I meant cmd tabbing. Application switching. Sorry -- was
born a Windows user, so used to calling it alt-tabbing.

Jeff
